### DISCUSSION TITLE
Change full screen on app launch

### DISCUSSION THREAD

**Discussion by lambaradan** [zed-industries/zed#9380] [Feature Requests]
The default seems to be full screen on launching the app. How can one choose a smaller size than full screen as the default?

**Comment by SomeoneToIgnore** [comment]
That is presumably fixed with https://github.com/zed-industries/zed/pull/9358 which will ship in Zed 0.128.0

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
